### PR TITLE
 [REF] import-checkers: Use of isort

### PR DESCRIPTION
**Pull request by moylop260**
- [x] Migrate imports to isort
 - [x] Add known libraries options.
 - [x] Fix red
 - [x] Add new isort package for setup depends
 - [x] Fix https://github.com/PyCQA/pylint/issues/875
 - [x] Fix https://github.com/PyCQA/pylint/issues/876

### CODE REVIEW

**Comment by moylop260** [comment]
PR ready to review

**Comment by moylop260** [comment]
@PCManticore 
What do you think about?

File: pylint/checkers/imports.py [hunk 1]
```diff
@@ -483,36 +506,34 @@ def _check_imports_order(self, node):
...
+            elif import_category == 'LOCALFOLDER':
+                local_imports.append((node, package))
             else:
-                try:
-                    filename = file_from_modpath([package])
-                except ImportError:
-                    continue
-                if not filename:
-                    continue
+                raise ValueError('Bad import category %s', import_category)
```
Comment (PCManticore) [inline pylint/checkers/imports.py]: Don't raise this, pylint shouldn't crash with these errors.

**Comment by PCManticore** [comment]
Apart of the small comment I left, looks good.

**Comment by moylop260** [comment]
@PCManticore 
Thanks for feedback

Fixed
- [x] [REF] wrong-import-order: Remove raise

**Comment by PCManticore** [comment]
Can you solve the conflicts, please? I would do it myself, but there is an oddity with your pull requests, I don't know in which repository they live, at least the Github UI provides only the text name, not the link to the repo.

**Comment by moylop260** [comment]
@PCManticore 
Don't worry

Fix
- [ ] Rebased

**Pull request merged**